src/main/java/org/apache/commons/lang3/LocaleUtils.java

```diff
@@ -87,6 +87,7 @@ public LocaleUtils() {
      * @throws IllegalArgumentException if the string is an invalid format
      * @see Locale#forLanguageTag(String)
      */
+    @SuppressWarnings("index:argument.type.incompatible") // #1: str.length > 3 as checked by #0.1
```

Comment:
Since that throws an illegal argument exception, I would annotate `str` as `@MinLen(3)`